### DISCUSSION TITLE
IDE-Roberto Lopez

### DISCUSSION THREAD

**Discussion by Luciano-M-Inacio** [ivanilmarcelino/designer#4] [Q&A]
Bom dia, eu uso a IDE do Roberto Lopez  a muito tempo, gostaria de saber se tem como converter os form´s da HMG para Minigui Extended?
Tenho muitos form´s para converter.
Obrigado.

**Comment by ivanilmarcelino** [comment]
Ola Luciano,

Sugiro entrar para o grupo onde pode ter o compartilhamento dos colegas que usam o Designer, aqui é muito raro eu ver pois tudo é tratado automaticamente, acesso raramente o git dessa forma;
De qualquer forma, acredito que formulários do HMG para a Extend deve abrir naturalmente, o contrário pode encontrar problemas por conta dos objetos que só tem na extend;
Espero ter ajudado.
